src/test/java/org/ambraproject/rhino/model/article/ArticleMetadataTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2017 Public Library of Science
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+package org.ambraproject.rhino.model.article;
+
+import com.google.gson.Gson;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class ArticleMetadataTest {
+
+  @Test
+  public void testSerialization() throws Exception{
+    //confirm that eIssn is not serialized to lowercase eissn
+    ArticleMetadata am = ArticleMetadata.builder().seteIssn("12345").build();
+    String amJson = new Gson().toJson(am);
+    assertEquals(amJson, "{\"eIssn\":\"12345\"}");
```

Comment:
:+1:

Reply:
although technically this is an invalid ISSN https://en.wikipedia.org/wiki/International_Standard_Serial_Number

Reply:
+1